pom.xml

```diff
@@ -286,6 +292,19 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```

Comment:
Could you please confirm that you shouldn't be using this instead: https://mvnrepository.com/artifact/org.flywaydb/flyway-maven-plugin/7.3.2